core/core.js

```diff
@@ -155,6 +155,10 @@ SharedQueue Binary Layout
     asyncHandlers[opId] = cb;
   }
 
+  function setAsyncHandlerByName(opName, cb) {
+    setAsyncHandler(opsCache[opName], cb);
+  }
```

Comment:
I think this function is superfluous

Reply:
I agree, but this is the simplest way to do use the `opsCache` in 10_dispatch_buffer.js.
Currently (on master), this opId -> opName mapping is setup in `99_main.js` for minimal ops.
In #9457 this is resolved/ removed (as all logic moves to core.js).

Reply:
Okay, didn't catch that, sounds reasonable then